sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -424,3 +424,26 @@ function buildRawAuthorizationRule(authorizationRule: AuthorizationRule): any {
   };
   return rawAuthorizationRule;
 }
+
+/**
+ * @ignore
+ * Helper utility to extract the relative path from given URL,
+ * or undefined / empty if not existing.
+ * Valid relative paths used in the ATOM based management API are in format
+ *     - `<queue-name>`
+ *     - `<topic-name>`
+ *     - `<topic-name>/Subscriptions/<subscription-name>`
+ *     - `<topic-name>/Subscriptions/<subscription-name>/Rules/<rule-name>`
+ * @param url
+ */
+export function getRelativePathOrUndefined(url: string): string | undefined {
```

Comment:
What happens when we return undefined? Will the value be ignored when the request to the service is constructed?

Reply:
Yes, undefined inputs to this property from user gets ignored. 
This utility here parses the result returned by service and has to do with response than request - and for the response as well, if it is undefined then we return it as undefined.